~~~
Command Output (stderr):
--
/Users/ec2-user/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift/test/Interop/Cxx/stdlib/print-libcxx-module-interface.swift:6:16: error: CHECK-NEXT: is not on the line after the previous match
// CHECK-NEXT: enum __1 {
               ^
<stdin>:141:2: note: 'next' match was here
 enum __1 {
 ^
<stdin>:100:11: note: previous match ended here
enum std {
          ^
<stdin>:101:1: note: non-matching line after previous match is here
 struct exception {
^

Input file: <stdin>
Check file: /Users/ec2-user/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift/test/Interop/Cxx/stdlib/print-libcxx-module-interface.swift

-dump-input=help explains the following input dump.

Input was:
<<<<<<
        .
        .
        .
      136:  struct __throw_with_nested<_Tp, _Up, > { 
      137:  } 
      138:  @available(*, unavailable, message: "Un-specialized class templates are not currently supported. Please use a specialization of this type.") 
      139:  struct __can_dynamic_cast<_From, _To> { 
      140:  } 
      141:  enum __1 { 
next:6      !~~~~~~~~~  error: match on wrong line
~~~

--- a/test/Interop/Cxx/stdlib/print-libcxx-module-interface.swift
+++ b/test/Interop/Cxx/stdlib/print-libcxx-module-interface.swift
@@ -3,7 +3,7 @@
 // REQUIRES: OS=macosx
 
 // CHECK: enum std {
-// CHECK-NEXT: enum __1 {
+// CHECK: enum __1 {
 
 // CHECK: typealias string =
 

--- a/test/Interop/Cxx/symbolic-imports/print-libcxx-symbolic-module-interface.swift
+++ b/test/Interop/Cxx/symbolic-imports/print-libcxx-symbolic-module-interface.swift
@@ -8,7 +8,7 @@
 // REQUIRES: OS=macosx
 
 // CHECK: enum std {
-// CHECK-NEXT: enum __1 {
+// CHECK: enum __1 {
 
 // STRING: struct basic_string {
 

